src/os/api/lib_io.lua

```diff
@@ -0,0 +1,88 @@
+function modular_computers.os.luaopen_io(stdin, stdout, stderr)
```

Comment:
The os namespace is reserved for overriding the lua api with the same name.
Please use the "internal" namespace, if its an internal function.

Reply:
I'm not sure. This particular function creates a library that is similar to Lua's `io` library (hence the name of the function) based on the stdin, stdout, and stderr streams passed to the function.